gen_code_can_common.c

```diff
@@ -0,0 +1,820 @@
+                           uint16_t main_v, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!altimeter) { return false; } 
+    if (!drogue_v) { return false; } 
+    if (!main_v) { return false; } 
+
+
+}
+
+int get_general_board_status(const can_msg_t *msg,
+                        )
+
+{
+    if (!msg){ return false; }
+    
+
+}
+
+bool get_sensor_temp(const can_msg_t *msg,
+                        uint8_t sensor_id, 
+                           uint24_t temperature, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!sensor_id) { return false; } 
+    if (!temperature) { return false; } 
+
+
+}
+
+bool get_sensor_altitude(const can_msg_t *msg,
+                        uint32_t altitude, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!altitude) { return false; } 
+
+
+}
+
+bool get_sensor_acc(const can_msg_t *msg,
+                        uint16_t x, 
+                           uint16_t y, 
+                           uint16_t z, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!x) { return false; } 
+    if (!y) { return false; } 
+    if (!z) { return false; } 
+
+
+}
+
+bool get_sensor_acc2(const can_msg_t *msg,
+                        uint16_t x, 
+                           uint16_t y, 
+                           uint16_t z, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!x) { return false; } 
+    if (!y) { return false; } 
+    if (!z) { return false; } 
+
+
+}
+
+bool get_sensor_gyro(const can_msg_t *msg,
+                        uint16_t x, 
+                           uint16_t y, 
+                           uint16_t z, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!x) { return false; } 
+    if (!y) { return false; } 
+    if (!z) { return false; } 
+
+
+}
+
+bool get_sensor_mag(const can_msg_t *msg,
+                        uint16_t x, 
+                           uint16_t y, 
+                           uint16_t z, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!x) { return false; } 
+    if (!y) { return false; } 
+    if (!z) { return false; } 
+
+
+}
+
+bool get_sensor_analog(const can_msg_t *msg,
+                        uint16_t value, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!value) { return false; } 
+
+
+}
+
+bool get_gps_timestamp(const can_msg_t *msg,
+                        uint8_t hrs, 
+                           uint8_t mins, 
+                           uint8_t secs, 
+                           uint8_t dsecs, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!hrs) { return false; } 
+    if (!mins) { return false; } 
+    if (!secs) { return false; } 
+    if (!dsecs) { return false; } 
+
+
+}
+
+bool get_gps_latitude(const can_msg_t *msg,
+                        uint8_t degs, 
+                           uint8_t mins, 
+                           uint16_t dmins, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!degs) { return false; } 
+    if (!mins) { return false; } 
+    if (!dmins) { return false; } 
+
+
+}
+
+bool get_gps_longitude(const can_msg_t *msg,
```

Comment:
name: get_gps_long
params: missing `uint8_t direction`, not shortened in actual (ie. `degrees` instead of `degs`)
missing:
```
    if (!direction) { return false; }
    if (get_message_type(msg) != MSG_GPS_LONGITUDE) { return false; }

    *degrees = msg->data[3];
    *minutes = msg->data[4];
    *dminutes = msg->data[5] << 8 | msg->data[6];
    *direction = msg->data[7];

    return true;
```